schemas/documentation_generation.py

```diff
@@ -58,6 +62,20 @@ class FirestoreDocumentationUpdateModel(BaseModel):
     relative_path: str
     usage: CompletionUsage
 
+class FirestoreRepoDocModel(BaseModel):
+    id: str
+    name: str
+    path: str
+    status: DocsStatusEnum
+    type: RepoNodeType
+
+class FirestoreRepoCreateModel(BaseModel):
+    dependencies: dict[str, str]
+    root_doc: FirestoreRepoDocModel       # {id: "doc_id_1", path: "/README.md", status: "COMPLETED"}
+    docs: list[FirestoreRepoDocModel]     # [{id: "doc_id_1", path: "/README.md", status: "COMPLETED"}, {id: "doc_id_2", path: "/README2.md", status: "STARTED"}]
+    version: str # commitId
+    repo_name: str
```

Comment:
is this going to be the repo's id?

Reply:
are you referring to the repo_name field? if so, i thought that would be the repo name like ours is rocketdocs. but maybe we should also return id?

Reply:
i think we should return the `id` also yeah